packages/sewing-kit-koa/src/middleware.ts

```diff
@@ -37,6 +37,7 @@ export default function middleware({
       assetPrefix,
       userAgent: ctx.get(Header.UserAgent),
       manifestPath,
+      countryCode: ctx.get('X-Ip-Metadata'),
```

Comment:
Assuming it will be the same as the header on core, `X-Ip-Metadata` will not be simply the country code but a string containing many things, including the country code.

Something like this:

```
asn:16276;anon:0;anonVpn:0;hostProvider:0;pubProxy:0;torExit:0;countryCode:CA
```

Here's the [code in storefront-renderer](https://github.com/Shopify/storefront-renderer/blob/eb55c8d6a8553bd11f4f4ad5c9abef3220195784/app/middlewares/ip_metadata_middleware.rb#L6) which parses out the country code from it.